extensions/Worker.Extensions.Http.AspNetCore/src/Coordinator/ContextReference.cs

```diff
@@ -32,10 +33,12 @@ public ContextReference(string invocationId)
 
         public TaskCompletionSource<FunctionContext> FunctionContextValueSource { get => _functionContextValueSource; set => _functionContextValueSource = value; }
 
-        internal void SetCancellationToken(CancellationToken token)
+        internal void SetCancellationToken(CancellationToken contextToken, CancellationToken invocationToken)
```

Comment:
Wonder if we can just use a `Task` instead of `CancellationToken` for tracking the invocation, considering we only care about completion.